src/components/TextInput/TextInput.module.scss

```diff
@@ -46,3 +46,25 @@
   background-color: var(--GrayStrokeAndDisabled--translucent);
   border-color: transparent;
 }
+
+.TextInputWrapper{
+  position: relative;
+}
+
+.TextInputIconWrapper{
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: 16px;
+}
+
+.TextInputIcon{
+  width:14px;
+  height:18px;
```

Comment:
nit pick. There should be spaces between property name and its value.

Reply:
and space between class name and `{`

Reply:
Fixed. Thank you